### PR TITLE
Fix teambuilder for LC formats

### PR DESCRIPTION
**Pull request by KrisXV**

### CODE REVIEW

**Comment by TheImmortal** [comment]
Instead of includes, what if you checked startsWith or endsWith? There’s probably some metagame out there with the letters lc, or one in the future, but if the formats starts or ends with LC then it’s likely an LC meta.

File: js/search.js [hunk 1]
```diff
@@ -849,6 +849,7 @@
 			else if (format === 'pu') tierSet = tierSet.slice(slices.PU || slices.NU);
 			else if (format === 'zu') tierSet = tierSet.slice(slices.ZU || slices.PU || slices.NU);
 			else if (format === 'lc' || format === 'lcuu') tierSet = tierSet.slice(slices.LC);
+			else if (format.includes('lc')) tierSet = tierSet.slice(slices["LC Uber"]);
```
[the remark this answers is not in the document]
Comment (KrisXV) [reply js/search.js]: a few lines up there is ``"Restricted Legendary"``

**Comment by KrisXV** [comment]
done @TheImmortal

**Comment by Zarel** [comment]
Exactly which LC formats is the current check designed for?

**Comment by TheImmortal** [comment]
LC versions of OMs.

**Comment by Zarel** [comment]
Can you give examples of ones that end with LC?

**Comment by TheImmortal** [comment]
Mix and Mega LC, LC AG.

**Pull request merged**